interpreter/cling/include/cling/Interpreter/InvocationOptions.h

```diff
@@ -70,9 +70,6 @@ namespace cling {
     /// \brief Architecture level of the CUDA gpu. Necessary for the
     /// NVIDIA fatbinary tool.
     std::string CUDAGpuArch;
-    /// \brief Contains arguments, which will passed to the nvidia tool
-    /// fatbinary.
-    std::vector<std::string> CUDAFatbinaryArgs;
```

Comment:
@SimeonEhrig, could you take a look?

Reply:
Sorry for the late reply. It is fine. I missed this in PR https://github.com/root-project/cling/pull/284/